src/Momento.Sdk/CacheClient.cs

```diff
@@ -1060,19 +1060,24 @@ public async Task<CacheListPopBackResponse> ListPopBackAsync(string cacheName, s
     }
 
     /// <inheritdoc />
-    public async Task<CacheListFetchResponse> ListFetchAsync(string cacheName, string listName)
+    public async Task<CacheListFetchResponse> ListFetchAsync(string cacheName, string listName, int? startIndex, int? endIndex)
```

Comment:
so here you are specifying these as nullable, but you are not providing default values.  In the interface, you are providing default values.  i'm a little surprised that works?  Does C# implicitly default nullable values to `null`?

I would think that without the `= null`s here in this signature that you wouldn't be able to call this method without explicitly passing the nulls in, which would be a breaking change... but it looks like the old tests that were calling this w/o these arguments are still passing, so I'm missing something somewhere :)

Reply:
My vote would be to make the default value explicit `= null` for both.

Reply:
I missed passing `null` there... so thank you for the call out!
I think it still works because I pass `null` in `ICacheClient` [here](https://github.com/momentohq/client-sdk-dotnet/pull/415/files#diff-9fc4e55974840be85a83f62e4c6a9e34615858796de63857408137cd172d5bc2R596)?